apps/projects/views.py

```diff
@@ -105,4 +106,9 @@ class ProjectListView(filter_views.FilteredListView):
     filter_set = ProjectFilterSet
 
     def get_queryset(self):
-        return super().get_queryset().filter(is_draft=False)
+        return super().get_queryset().filter(
+            Q(is_public=False, participants__pk=self.request.user.pk) |
```

Comment:
this will also show private projects, that are still in draft mode. is this intended?